src/outpace/config.clj

```diff
@@ -60,11 +76,39 @@
     (->EnvVal name (System/getenv name) (contains? (System/getenv) name))
     (throw (IllegalArgumentException. (str "Argument to #config/env must be a string: " (pr-str name))))))
 
-(defrecord FileVal [path contents exists?]
+(defonce etcd-prefix (atom nil))
+
+(defn etcd-get [name]
+  (etcd/get
+    (if @etcd-prefix
+      (str @etcd-prefix "/" name)
```

Comment:
Multiple derefing like this is not thread safe. The value of the atom could be changed between your derefs.